stock_move_line_multi_company_security/tests/test_stock_move_line_multi_company_security.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2018 Tecnativa S.L. - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.exceptions import AccessError
+from odoo.tests import common
+
+
+class TestStockMoveLineMultiCompany(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestStockMoveLineMultiCompany, cls).setUpClass()
```

Comment:
Well, you are here testing security rules, not the module itself, but well, as you have already developed them, it's good to have it to see if other module changes this.